docs/pbc/all/payment-service-providers/payone/manage-in-the-back-office/activate-payment-methods.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Activate payment methods
```

Comment:
We already have this page: [Edit payment methods](https://docs.spryker.com/docs/scos/user/back-office-user-guides/202204.0/administration/payment-methods/edit-payment-methods.html).